lib/ret_web/templates/page/robots.txt.eex

```diff
@@ -0,0 +1,13 @@
+<%=if @allow_crawlers do %>
+User-agent: *
+Allow: /
+<% else %>
+User-agent: *
+Disallow: /
+
+User-agent: Twitterbot
+Allow: /
+
+User-agent: facebookexternalhit
+Allow: /
```

Comment:
Do we need to allow slack and discord crawlers as well?

Reply:
afaict, no, slack explicitly says they ignore it, couldn't find anything on discord.